docs/workforce-management-framework/topics/con-wfm-framework-concepts.adoc

```diff
@@ -0,0 +1,31 @@
+[id='{context}-con-wfm-framework-concepts']
+= Introducing the {WFM-RC-NameShort} Framework Concepts
+
+The *{WFM-RC-NameShort} Framework* includes three concepts:
+
+.{WFM-RC-NameShort} Framework Concepts
+|===
+
+|Concept |Overview |User Role
+
+|Step Definition
+|A _Step Definition_ is a predefined piece of functionality, for example, functionality that allows the user to capture a signature
+|A Developer will typically works with a _Step Definition_
+
+|Step Workflow
+|A _Step Workflow_ can contain one to many _Step Definitions_. It defines the order the _Step Definitions_ are executed and contains the logic for executing the _Step Definition_
+|A user using the Portal Application typically works with a _Step Workflow_
+
+|WorkOrder
+|A _WorkOrder_ is an instantiation of a _Step Workflow_ . Each time a user creates a new _Step Workflow_, it is instantiated as a new _Step Workflow_.
```

Comment:
Another suggestion: "a series of steps that must be completed by the assignee. A workorder is a workflow that has been assigned to an App user."